### DISCUSSION TITLE
Does openllm start support model sharding at inference?

### DISCUSSION THREAD

**Discussion by gbmarc1** [bentoml/OpenLLM#274] [Q&A]
I see that the `accelerate` library is in the dependencies. However, I cannot see any argument that enables [model sharing](https://medium.com/@jain.sm/sharding-large-models-for-parallel-inference-ee19844cc44) for inference. I may be missing something tho.

**Comment by aarnphm** [comment]
We currrently shard the model via `device_map`.

**Comment by aarnphm** [comment]
But we are working on an internal sharding feature right now, so stay tuned!
